test/core/rules/test_cases/L003.yml

```diff
@@ -1,24 +1,26 @@
 rule: L003
 
-test_1:
```

Comment:
Great tests! Love the new ones, the names, and the comments. My 🧠 is so happy now, ha ha.